sources/packages/backend/apps/api/src/main.ts

```diff
@@ -55,6 +55,7 @@ async function bootstrap() {
   app.enableCors({
     exposedHeaders: "Content-Disposition",
     origin: allowAnyOrigin,
+    credentials: true,
```

Comment:
Asking for my understanding. Why do we need to enable this cors header?

Reply:
This will allow the client (Axios) to process the 'set-cookie' header and persist the cookie to the browser.